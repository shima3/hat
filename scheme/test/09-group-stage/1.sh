#!/bin/bash
$* examples/GroupStage.sch
