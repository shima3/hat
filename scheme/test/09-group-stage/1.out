start
R12 J7 S7 C3 P0
end
